tembo-stacks/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "tembo-stacks"
 description = "Tembo Stacks for Postgres"
-version = "0.17.2"
+version = "0.17.3"
```

Since we are adding a completely new feature but still backwards compatible.

```suggestion
version = "0.18.0"
```